### PR TITLE
Make Resources service an ampdoc-level service

### PR DESCRIPTION
**Pull request by dvoytenko**
Partial for #3742.

### CODE REVIEW

**Comment by aghassemi** [comment]
LGTM but please rebase, there is a `resourcesFor` left in `amp-lightbox-viewer.js` as well.

**Comment by aghassemi** [comment]
LGTM

File: testing/iframe.js [hunk 1]
```diff
@@ -198,9 +202,12 @@ export function createIframePromise(opt_runtimeOff, opt_beforeLayoutCallback) {
       if (opt_runtimeOff) {
         iframe.contentWindow.name = '__AMP__off=1';
       }
-      installDocService(iframe.contentWindow, true);
+      const ampdocService = installDocService(iframe.contentWindow, true);
+      const ampdoc = ampdocService.getAmpDoc(iframe.contentWindow.document);
       installExtensionsService(iframe.contentWindow);
       installRuntimeServices(iframe.contentWindow);
+      installAmpdocServices(ampdoc);
+      debugger;
```
Comment (jridgewell) [inline testing/iframe.js]: Hey there.

**Pull request merged**